elixir/lib/contracts.ex

```diff
@@ -25,20 +25,19 @@ defmodule Trento.Contracts do
         DateTime.utc_now()
       )
 
-    time_attr = Google.Protobuf.Timestamp.new!(seconds: time |> DateTime.to_unix())
```

Comment:
With this change we don't have validation I guess.
We don't we go with `struct!` which is the 1:1 replacement for `new!`